Source/Scene/Scene.js

```diff
@@ -3461,13 +3461,6 @@ function updateAndClearFramebuffers(scene, passState, clearColor) {
     environmentState.useOIT = oit.isSupported();
   }
 
-  if (
```

Comment:
This change stands out a bit but I believe it is fine because the clears will now always happen when `TranslucentTileClassification.execute` is called.